server/data/incidentReportingApi.ts

```diff
@@ -122,6 +122,20 @@ export type PaginationSortingParams = {
   sort: string[]
 }
 
+export type Question = {
+  code: string
+  question: string
+  responses: Response[]
+  additionalInformation: string | null
+}
+
+export type Response = {
```

Comment:
`Response` is a potentially bad/sneaky name because of other meanings/classes (e.g. `superagent` also has a `Response` type)